wagtail/contrib/search_promotions/migrations/0003_auto_20180524_1517.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.5 on 2018-05-24 15:17
+from __future__ import unicode_literals
```

Comment:
Minor additional item - Django 1.x is no longer supported, nor is Python 3 so this could be revised to be simpler (code below), also it would be good to name the migration file with something easier to understand at first glance e.g. `0003_add_external_urls.py`.

```
# -*- coding: utf-8 -*-
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):
```